docs/services/influxdb.md

```diff
@@ -0,0 +1,42 @@
+# InfluxDB
+
+[InfluxDB](https://www.influxdata.com/) is a self-hosted time-series database, that this playbook can install, powered by the [mother-of-all-self-hosting/ansible-role-influxdb](https://github.com/mother-of-all-self-hosting/ansible-role-influxdb) Ansible role.
+
+## Configuration
+
+To enable this service, add the following configuration to your `vars.yml` file and re-run the [installation](../installing.md) process:
+
+```yaml
+########################################################################
+#                                                                      #
+# influxdb                                                           #
+#                                                                      #
+########################################################################
+
+# Configuring this role for your playbook
+
+influxdb_enabled: true
+influxdb_hostname: 'example.org'
+
+
+# Advanced configuration
+# To bootstrap an initial user, bucket and organization you can uncomment and replace
+
+#influxdb_init: true
+#influxdb_init_username: "USERNAME"
+#influxdb_init_password: "SUPERSECRETPASSWORD"
+#influxdb_init_org: "EXAMPLE_ORG"
+#influxdb_init_bucket: "SOMEBUCKET"
```

Comment:
If this is enabled:

1. Is it safe to leave it in `vars.yml` for subsequent runs? Or will it break?
2. If you change the credentials subsequently, will it create yet another user or does it only work the first time around?

We should probably put the answers to these questions both here and inside `default/main.yml`.

Reply:
1. Yes, it's save
2. This is only used for initial setup and will not change on subsequent runs (specifically this setting has no effect if an existing boltdb is found)